app/models/manageiq/providers/google/manager_mixin.rb

```diff
@@ -19,6 +23,15 @@ def gce
     @gce ||= connect(:service => "compute")
   end
 
+  private
+
+  def verify_pubsub_credentials(options = {})
+    connect(options.merge(:service => "pubsub")).subscriptions.all
```

Comment:
Should the `!!` be here instead of up above?

Reply:
Should this pass the validate param as true? also like it does on line 6?

Reply:
> Should this pass the validate param as true also like it does on line 6?

No, because that validation calls `.regions.all` which isn't valid on the pubsub service